src/Model/ApplicationSchema.php

```diff
@@ -6,7 +6,7 @@
  *
  * @category Class
  * @package  RusticiSoftware\Cloud\V2
- * @author   Swagger Codegen team
+ * @author   Swaagger Codegen team
```

Comment:
Not sure how the extra "a" got in here.

That being said, shouldn't the author for these files actually be something like `Rustici Software`, even though we're generating the code?